data/fin.lean

```diff
@@ -88,3 +91,23 @@ by cases i; refl
   {n} {C : fin (succ n) → Sort*} {H0 Hs} (i : fin n) :
   @fin.cases n C H0 Hs i.succ = Hs i :=
 by cases i; refl
+
+def nat_expand_type {n} (i : fin n) (k) : fin (n + k) :=
+⟨i.val,
+begin
```

Comment:
This proof can be written more concisely as `lt_add_of_lt_of_nonneg i.2 (nat.zero_le k)`

Reply:
@cipher1024 Don't do it all yourself! 😜

Reply:
Applied.